.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,11 @@
+blank_issues_enabled: false
+contact_links:
+    - name: Main React Native Repository
```

Comment:
Let's add a link to the Expo issue tracker here:
https://github.com/expo/expo/issues/new/choose